cfme/utils/appliance/__init__.py

```diff
@@ -853,6 +852,10 @@ def ssh_client_with_privatekey(self):
         store.ssh_clients_to_close.append(ssh_client)
         return ssh_client
 
+    @cached_property
```

Comment:
**required** this is a alias, we shouldn't cache it, but instead try to pass trough, i presume there are no more sites that set it?